napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -58,6 +61,23 @@
 InstallerTypes = Literal['pip', 'mamba']
 
 
+def _pip_constraints():
+    return [f"napari=={__version__}", "pydantic<2"]
+
+
+def _conda_constraints():
+    return [f"napari={__version__}", "pydantic<2.0a0"]
```

Comment:
This is better handled in a repodata patch (pip doesn't have this mechanism but conda-forge does). Now this constraint will be tied to that release forever, which is maybe not a problem in this case, but we need to be careful about internalizing "external" problems in our codebase.

Reply:
Looks like `_conda_constraints` is not used anywhere anyway, so I'll delete it in the [now-necessary follow-up PR](https://forum.image.sc/t/request-for-testing-napari-0-4-18-release-candidate/82833/21).